tests/system/load/test_llm.py

```diff
@@ -54,7 +54,7 @@ def test_llm_palm_configure_fit(llm_fine_tune_df_default_index, llm_remote_text_
         model_name="text-bison", max_iterations=1
     )
 
-    df = llm_fine_tune_df_default_index.dropna()
+    df = llm_fine_tune_df_default_index.dropna().sample(n=100)
```

Comment:
curious how much run time reduction with sampling.

Reply:
It didn't improve much. It still takes the same amount of time around 30-40 mins, and the durations has some variations.